### PR TITLE
Fixed #53: Added icons and fixed links

### LINKED ISSUE #53: bugged links and icons
The links for GitHub, twitter and facebook don't work and there is just plain text instead of icons. The GitHub icon does not even have plain text
I am working on it, pls take my consent before proceeding with this issue

### PR DESCRIPTION
**Pull request by samyakbvs**
Added Icons instead of plain text and fixed the bugged link for github

### CODE REVIEW

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyak--2014.fossasia.org/

File: _includes/footer.html [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-	<footer style="color: black">
+	<footer style="padding-top: 20px;" align="center" style="color: black">
+		
 		<font color="white">
-			<a href="facebook.com/fossasia">Facebook</a>
-			<a href="twitter.com/fossasia">Twitter</a>
-			<a href="github.com/fossasia"></a>
+		<div class="container">
+			<a href="https://www.facebook.com/fossasia/"><i align="left" style="padding: 3px; color: #000000; font-size: xx-large;" class="fa fa-facebook-official" aria-hidden="true"></i></a>
+			<a href="https://twitter.com/fossasia?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"><i align="left" style="padding: 3px; color: #000000; font-size: xx-large;" class="fa fa-twitter-square" aria-hidden="true"></i></a>
```
Comment (0x48piraj) [inline _includes/footer.html]: It's a bad link, consisting of several `ref` redirects. fix them.

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
See the code review. Cheers :+1:

File: _includes/footer.html [hunk 2]
```diff
@@ -1,11 +1,14 @@
-
+</div>
 	</div>
-	</div>
-	<footer style="color: black">
+	<footer style="padding-top: 20px;" align="center" style="color: black">
+
```
Comment (abishekvashok) [inline _includes/footer.html]: You got some tabs here. Remove em pls.

File: _includes/footer.html [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
 	</div>
-	</div>
-	<footer style="color: black">
+	<footer style="padding-top: 20px;" align="center" style="color: black">
+		
 		<font color="white">
-			<a href="facebook.com/fossasia">Facebook</a>
-			<a href="twitter.com/fossasia">Twitter</a>
-			<a href="github.com/fossasia"></a>
+		<div class="container">
```
Comment (abishekvashok) [inline _includes/footer.html]: Please properly intend

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by samyakbvs** [comment]
Changes made @abishekvashok

**Review by harshit98 — APPROVED** [review]

**Comment by abishekvashok** [comment]
Good job

**Pull request merged**